src/popup/components/Modals/TransferReceive.vue

```diff
@@ -15,6 +15,8 @@
 
       <AccountRow />
 
+      <div style="height: 400px;" />
```

Comment:
Why did you start to use `style` attribute?

Reply:
This was an element that allowed me to test scrolling on the modals. I removed this.